docs/how-to-guides/monitoring-with-prometheus-operator.md

```diff
@@ -0,0 +1,135 @@
+# Setting up monitoring on Lokomotive
+
+## Contents
+
+* [Introduction](#introduction)
+* [Prerequisites](#prerequisites)
+* [Deploy Prometheus Operator](#deploy-prometheus-operator)
+  * [Configure Prometheus Operator](#configure-prometheus-operator)
+  * [Install Prometheus Operator](#install-prometheus-operator)
+* [Accessing Prometheus, Alertmanager and Grafana](#accessing-prometheus-operator-sub-components)
+  * [Accessing Prometheus](#accessing-prometheus)
+  * [Accessing Alertmanager](#accessing-alertmanager)
+  * [Accessing Grafana](#accessing-grafana)
+    * [Using port forward](#using-port-forward)
+    * [Using Ingress](#using-ingress)
+* [Additional resources](#additional-resources)
+
+## Introduction
+
+This guide provides the steps for deploying a monitoring stack using the `prometheus-operator` Lokomotive component and explains how to access Prometheus, Alertmanager and Grafana.
+
+## Prerequisites
+
+* A Lokomotive cluster deployed on a supported provider and accessible via `kubectl`.
+
+<!---
+TODO: Once we have tutorials on how to deploy and configure Rook or OpenEBS, point the following to those tutorials.
+-->
+
+* A storage provider component (`rook` and `rook-ceph` or `openebs-operator` and `openebs-storage-class`) deployed with a default storage class that can provision volumes for the [PVCs](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims) created by Alertmanager and Prometheus.
+
+> **NOTE**: If you wish to [expose Grafana to the public internet](#using-ingress), the following Lokomotive components should be installed:
+
+<!---
+TODO: Once we have a tutorial on how to deploy and configure Contour and cert-manager, point the following to that tutorial.
+-->
+
+  * [`metallb`](./ingress-with-contour-metallb.md) (only on Packet and bare-metal)
+  * [`contour`](../configuration-reference/components/contour.md)
+  * [`cert-manager`](../configuration-reference/components/cert-manager.md)
```

Comment:
This looks a bit weird in rendering:
![image](https://user-images.githubusercontent.com/507354/85845904-4bf42600-b7a5-11ea-8cbb-b1cf8d2be56e.png)


Can we make it all in quotation format? I think this should do the trick

```suggestion
<!---
TODO: Once we have a tutorial on how to deploy and configure Contour and cert-manager, point the following to that tutorial.
-->

> **NOTE**: If you wish to [expose Grafana to the public internet](#using-ingress), the following Lokomotive components should be installed:
> * [`metallb`](./ingress-with-contour-metallb.md) (only on Packet and bare-metal)
> * [`contour`](../configuration-reference/components/contour.md)
> * [`cert-manager`](../configuration-reference/components/cert-manager.md)
```